docs/api.md

```diff
@@ -16,12 +16,17 @@ These are the API group versions which are curently available when using Antrea.
 | `clusterinformation.antrea.tanzu.vmware.com` | `v1beta1` | No | v0.3.0 | N/A | N/A |
 | `core.antrea.tanzu.vmware.com` | `v1alpha2` | No | v0.11.0 | N/A | N/A |
 | `controlplane.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.10.0 | v0.11.0 | Aug 2021 |
-| `controlplane.antrea.tanzu.vmware.com` | `v1beta2` | Yes | v0.11.0 | N/A | N/A |
+| `controlplane.antrea.tanzu.vmware.com` | `v1beta2` | Yes | v0.11.0 | v1.0.0 | N/A |
```

Comment:
you can set the "Planned Removal" value for all of these to `Dec 2021`

Reply:
Got it, thanks.